Change icon to a blown up AppCenter

hanaral opened the pull request.

pongloongyeat commented:
This is pretty nice :) I'm not sure if the AppCenter icon shows that processes/applications are getting killed though :( but it definitely looks better than mine (I'm horrible at iconography)

Could you move this into a folder called `icons/` along with the current icon.svg?

hanaral commented:
Thanks! I added the folder and updated the README to use the new location
The reason I chose the AppCenter icon was because it is the only 'correct' way you get apps - and the only official part of the branding icons that is remotely to do with applications (its supposed to be like the broken Finder icon killall has)

pongloongyeat commented:
Looks good! Thanks for the icon again!
